frontend/src/components/Workspace/Visualize/VisualizeItem.tsx

```diff
@@ -60,8 +62,12 @@ export const VisualizeItem = memo(function VisualizeItem({
   )
   const { size, onMouseDownX, onMouseDownY, onMouseDownXY } =
     useItemDragResize(itemId)
+  const loading = useSelector(selectLoadingVisualize)
   return (
-    <Box sx={{ m: 1, display: "flex", flexDirection: "row" }}>
+    <Box
+      sx={{ m: 1, display: "flex", flexDirection: "row", position: "relative" }}
+    >
+      {loading ? <Loading position={"absolute"} /> : null}
```

Comment:
with loading on this component, all other plots show loading.
I would like the loading to be displayed only for the plot that is editing the ROI.

Reply:
@ReiHashimoto I added condition to check loading, pls check it again